dev/.rat-excludes

```diff
@@ -101,3 +101,4 @@ org.apache.spark.scheduler.ExternalClusterManager
 .*\.sql
 .Rbuildignore
 org.apache.spark.deploy.yarn.security.ServiceCredentialProvider
+spark-warehouse
```

Comment:
oh whoops, I must of committed that by accident, dev/run-tests won't pass without it so I've been adding/removing it every time